app/src/routes/v1/users.js

```diff
@@ -28,4 +28,28 @@ usersRouter.get('/:keycloakId/acronyms', [
   }
 });
 
+/** Returns acronym service clients associated with a user */
+usersRouter.get('/:keycloakId/acronyms/clients', [
+  param('keycloakId', 'Must be a valid UUID')
+    .exists().custom(value => validator.isUUID(value))
```

Comment:
We actually don't want the .exists() as it will cause a double report in the errors array.

Reply:
kk